Client/Frontend/Fakespot/Components/CollapsibleCardContainer.swift

```diff
@@ -0,0 +1,169 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Common
+import UIKit
+
+class CollapsibleCardContainer: CardContainer, UIGestureRecognizerDelegate {
+    private struct UX {
+        static let verticalPadding: CGFloat = 8
+        static let horizontalPadding: CGFloat = 8
+        static let titleHorizontalPadding: CGFloat = 16
+        static let titleTopPadding: CGFloat = 16
+        static let chevronSize = CGSize(width: 20, height: 20)
+    }
+
+    enum ExpandButtonState {
+        case collapsed
+        case expanded
+
+        var image: UIImage? {
+            switch self {
+            case .expanded:
+                return UIImage(named: StandardImageIdentifiers.Large.chevronUp)?.withRenderingMode(.alwaysTemplate)
+            case .collapsed:
+                return UIImage(named: StandardImageIdentifiers.Large.chevronDown)?.withRenderingMode(.alwaysTemplate)
+            }
+        }
+
+        var toggle: ExpandButtonState {
+            switch self {
+            case .expanded:
+                return .collapsed
+            case .collapsed:
+                return .expanded
+            }
+        }
+    }
+
+    // MARK: - Properties
+    private var state: ExpandButtonState = .expanded
+
+    // UI
+    private lazy var rootView: UIView = .build { _ in }
+    private lazy var headerView: UIView = .build { _ in }
+    private lazy var containerView: UIView = .build { _ in }
+    private var containerHeightConstraint: NSLayoutConstraint?
+    private var tapRecognizer: UITapGestureRecognizer!
+
+    lazy var titleLabel: UILabel = .build { label in
+        label.adjustsFontForContentSizeCategory = true
+        label.font = DynamicFontHelper.defaultHelper.preferredFont(withTextStyle: .headline, size: 17.0)
+        label.numberOfLines = 0
+    }
+
+    private lazy var chevronButton: ResizableButton = .build { view in
+        view.setImage(self.state.image, for: .normal)
+        view.buttonEdgeSpacing = 0
+        view.addTarget(self, action: #selector(self.toggleExpand), for: .touchUpInside)
+    }
+
+    // MARK: - Inits
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+
+        setupLayout()
+
+        tapRecognizer = UITapGestureRecognizer(target: self, action: #selector(tapHeader))
+        tapRecognizer.delegate = self
+        headerView.addGestureRecognizer(tapRecognizer)
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func configure(_ view: UIView) {
+        configure(title: "", contentView: view, titleA11yId: "", closeButtonA11yId: "", expandState: .collapsed)
```

Comment:
nit, I would add a comment here to explain context for this. It feels strange otherwise?
```suggestion
    override func configure(_ view: UIView) {
        configure(title: "", contentView: view, titleA11yId: "", closeButtonA11yId: "", expandState: .collapsed)
```